docs/whatsnew/7.1.rst

```diff
@@ -45,6 +45,52 @@ is now supported by the Unified File Read/Write Interface.
 Note that this format is compatible with ``Table`` objects with integer, float,
 and/or string data types, but not ``QTable`` objects or other data types.
 
+.. _whatsnew-7.1-cosmology:
+
+Cosmology
+=========
+
+
+The `astropy.cosmology.traits` module hosts various parts of cosmologies, such as the
+:class:`~astropy.cosmology.traits.ScaleFactor` or
+:class:`~astropy.cosmology.traits.TemperatureCMB`. These traits can be used to more
+easily construct custom cosmologies by combining different components.
+
+As a simple example, the :class:`~astropy.cosmology.traits.TemperatureCMB` trait
+provides the ``Tcmb0`` property and
```

Comment:
Add a link.

Reply:
Can't. This was the change necessary to get Sphinx to guild. Thankfully if someone clicks on `TemperatureCMB` it renders correctly.
https://github.com/astropy/astropy/pull/17702/commits/6d3d657e049d56d8cc4ea7fc97f51d4f0065c4e8

Reply:
Can't. This was part of the change necessary to get Sphinx to guild.
Thankfully if someone clicks on TemperatureCMB it renders correctly.
https://github.com/astropy/astropy/pull/17702/commits/6d3d657e049d56d8cc4ea7fc97f51d4f0065c4e8